### DISCUSSION TITLE
input declaration in sample_report process

### DISCUSSION THREAD

**Discussion by gregdenay** [bio-raum/FooDMe2#71] [General]
@marchoeppner while checking the problem with the crappy data I noticed that the input takes a tuple of files with metadata and a file with just the path:

```
    input:
    tuple val(meta), path(composition), path(cutadapt), path(blast), path(consensus), path(fastp)
    path(clustering)
```

Is this right? Shouldn't the `clustering` path be part of the tuple so samples don't get mixed up ?

**Comment by marchoeppner** [comment]
That file seems to contain results from all the samples (and is used as input to all the sample-level report processes, as defined by the .collect() method). 

And in the script, I see:

```PYTHON
        "clustering": cluster_dict["data"][sample_id],

```

So the script then seems to grab the results for the active samples. Should be fine.
